prbt_hardware_support/config/modbus_read_api_spec_pnoz.yaml

```diff
@@ -1,5 +1,10 @@
 # PNOZMulti Software V2.1dev
-STO: 512
-VERSION: 513
-BRAKETEST_REQUEST: 514
-OPERATION_MODE: 515
\ No newline at end of file
+# Generic: Enabled the whole time
```

Comment:
The purpose of this file is to define, under which register the Modbus adapter can read the different values.
It should not document some behaviour of how and when those are enabled or disabled.
It would maybe make sense to use these lines to generally describe what the register content describes or skip them altogether.

```suggestion
# STO: This register contains the state of the robots emergency stop signal
```